metricbeat/module/vsphere/host/data_host_system.go

```diff
@@ -0,0 +1,129 @@
+package host
```

Comment:
It looks like you could have got away with a much smaller set of stub data for your testing purposes because `eventMapping` only needs a few of the fields.